samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -550,12 +550,12 @@ static int configure_stream(struct bt_audio_stream **stream)
 {
 	int err;
 
-	*stream = bt_audio_stream_config(default_conn, sinks[0],
-					 remote_capabilities[0],
-					 &preset_16_2_1.codec);
-	if (*stream == NULL) {
+	err = bt_audio_stream_config(default_conn, &audio_stream, sinks[0],
```

Comment:
I guess that `configure_stream()`no longer needs to take a pointer to a pointer to a stream, but could take only a pointer to a stream?

Reply:
Correct, which is also what it does by `&audio_stream` as `audio_stream` is just a statically allocated struct. The function is updated later to use the `stream` from the parameter.

Reply:
I see what happened - I was commenting on a commit in the middle, where the function signature still had the pointer to pointer ...